active/corsair.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is an adaptation of Corsair Scan (https://github.com/Santandersecurityresearch/corsair_scan) as a ZAP active scan script.
+This script will resend requests to all the sites captured in ZAP, injecting different origins. Then, it will read the value of Access-Control-Allow-Origin and based on that, it
+will assess if CORS is properly configured.
+"""
+
+import urlparse
+alertTitle = 'Corsair - CORS Misconfigured.'
+alertDescription = "Cross Origin Resource Sharing (CORS) is misconfigured. \n"
+alertRisk = 2
+alertReliability = 3
+alertSolution = "Configure CORS in a more restrictive way, whitelisting only the sites allowed to access to your domain."
```

Comment:
We have been replacing whitelist, we should not introduce them. The last "to" does not seem to be needed.

Reply:
i mean whitelisting the domain in the CORS configuration. What would you say? Configure CORS in a more restrictive way, to give access only the sites allowed to access your domain. ?

Reply:
Yup that sounds good